pkg/apis/externaldns/types.go

```diff
@@ -29,57 +30,69 @@ var (
 type Config struct {
-	Master             string
-	KubeConfig         string
-	Sources            []string
-	Namespace          string
-	FQDNTemplate       string
-	Compatibility      string
-	PublishInternal    bool
-	Provider           string
-	GoogleProject      string
-	DomainFilter       []string
-	AWSZoneType        string
-	AzureConfigFile    string
-	AzureResourceGroup string
-	CloudflareProxied  bool
-	Policy             string
-	Registry           string
-	TXTOwnerID         string
-	TXTPrefix          string
-	Interval           time.Duration
-	Once               bool
-	DryRun             bool
-	LogFormat          string
-	MetricsAddress     string
-	LogLevel           string
+	Master               string
+	KubeConfig           string
+	Sources              []string
+	Namespace            string
+	FQDNTemplate         string
+	Compatibility        string
+	PublishInternal      bool
+	Provider             string
+	GoogleProject        string
+	DomainFilter         []string
+	AWSZoneType          string
+	AzureConfigFile      string
+	AzureResourceGroup   string
+	CloudflareProxied    bool
+	InfobloxGridHost     string
+	InfobloxWapiPort     int
+	InfobloxWapiUsername string
+	InfobloxWapiPassword string
+	InfobloxWapiVersion  string
+	InfobloxSSLVerify    bool
+	Policy               string
+	Registry             string
+	TXTOwnerID           string
+	TXTPrefix            string
+	Interval             time.Duration
+	Once                 bool
+	DryRun               bool
+	LogFormat            string
+	MetricsAddress       string
+	LogLevel             string
 }
 
 var defaultConfig = &Config{
-	Master:             "",
-	KubeConfig:         "",
-	Sources:            nil,
-	Namespace:          "",
-	FQDNTemplate:       "",
-	Compatibility:      "",
-	PublishInternal:    false,
-	Provider:           "",
-	GoogleProject:      "",
-	DomainFilter:       []string{},
-	AWSZoneType:        "",
-	AzureConfigFile:    "/etc/kubernetes/azure.json",
-	AzureResourceGroup: "",
-	CloudflareProxied:  false,
-	Policy:             "sync",
-	Registry:           "txt",
-	TXTOwnerID:         "default",
-	TXTPrefix:          "",
-	Interval:           time.Minute,
-	Once:               false,
-	DryRun:             false,
-	LogFormat:          "text",
-	MetricsAddress:     ":7979",
-	LogLevel:           logrus.InfoLevel.String(),
+	Master:               "",
+	KubeConfig:           "",
+	Sources:              nil,
+	Namespace:            "",
+	FQDNTemplate:         "",
+	Compatibility:        "",
+	PublishInternal:      false,
+	Provider:             "",
+	GoogleProject:        "",
+	DomainFilter:         []string{},
+	AWSZoneType:          "",
+	AzureConfigFile:      "/etc/kubernetes/azure.json",
+	AzureResourceGroup:   "",
+	CloudflareProxied:    false,
+	InfobloxGridHost:     "",
+	InfobloxWapiPort:     443,
+	InfobloxWapiUsername: "admin",
+	InfobloxWapiPassword: "",
+	InfobloxWapiVersion:  "2.3.1",
+	InfobloxSSLVerify:    false,
```

Comment:
Is this validating the infoblox API server cert? Because then we should probably enable it by default.

Reply:
Hi @linki, that is the intent of this flag, but an issue I encountered with `kingpin` was that I could toggle it on with `--infoblox-ssl-verify` but could not toggle it off with `--infoblox-ssl-verify=false`, and so set `false` as an unfortunate default.

Reply:
@khrisrichardson kingpin will automatically create --no-infoblox-ssl-verify for bool flags.

Reply:
Sweet. I'll go ahead and change the default.